src/main/java/com/sap/sgs/phosphor/fosstars/model/feature/DoubleFeature.java

```diff
@@ -0,0 +1,27 @@
+package com.sap.sgs.phosphor.fosstars.model.feature;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sap.sgs.phosphor.fosstars.model.Value;
+import com.sap.sgs.phosphor.fosstars.model.value.DoubleValue;
+
+/**
+ * A feature which holds a double.
+ */
+public class DoubleFeature extends AbstractFeature<Double> {
+
+  @JsonCreator
+  public DoubleFeature(@JsonProperty("name") String name) {
+    super(name);
+  }
+
+  @Override
+  public DoubleValue value(Double n) {
+    return new DoubleValue(this, n);
+  }
+
+  @Override
+  public Value<Double> parse(String string) {
+    return value(Double.parseDouble(string));
```

Comment:
Are you expecting NPEs or NumberFormat Exceptions or you would want to check the string better? )

Reply:
I would not like to catch this runtime exceptions. I would prefer to let `Double.parseDouble()` throw them according to its javadoc.

Reply:
No, I mean, do you want to check for the format of the String before atleast for a null? because NPEs doesn't look good in the logs.

Reply:
I understood what you mean. The `Double.parseDouble()` method expectes a null and throws an NPE if a string is null:

https://docs.oracle.com/javase/7/docs/api/java/lang/Double.html#parseDouble(java.lang.String)

I can add an `Objects.requireNotNull(string, ...)` which also throws an NPE if an object is null. The only thing we get here is a custom exception message. If you think it's worth it, I can add such a check. I don't think we should check the format of the string - the `parseDouble()` method is supposed to do that. We can only check for an empty string. If you think it's worth it, I can add such a check. There is no problem.

Reply:
How about IllegalArgumentException() with the message? :) If you think it's not needed, please go ahead, no worries at all )

Reply:
I think that a NullPointerException should be thrown if a string is null. IllegalArgumentException can be thrown if a string is empty. I've added the checks.

Reply:
Ok, thanks for adding it )